scripts/gen-global-typings.js

```diff
@@ -0,0 +1,25 @@
+// This script generates global.d.ts, a transformation of the type declarations
+// of the decorators module into the Polymer.decorators global namespace, which
+// corresponds to what our rollup configuration does.
+//
+// This script should typically be run with `npm run build`.
+
+const fs = require('fs');
+
+const input = fs.readFileSync('lib/decorators.d.ts').toString();
+const output = `// DO NOT EDIT
+// This file is automatically generated by \`npm run build\`.
+
+declare namespace Polymer {
+  namespace decorators {
+${
+    input
+        // We don't need to export or declare each member of the namespace.
+        .replace(/^export\s*/gm, '')
```

Comment:
Let's make these `\s+` to enforce the whitespace. This removes the possibility of potentially matching non-keyword exports.

Reply:
Done.